src/moin/macros/_base.py

```diff
@@ -85,13 +85,24 @@ def extract_h1(item_name):
     return _("No heading found in item: {item_name}").format(item_name=item_name)
 
 
-def fail_message(msg, severity="error"):
+def fail_message(msg, alternative, severity="error"):
     """
     Return an error message in admonition-like format.
 
-    Severity may be: attention, caution, danger, error, hint, important, note, or tip.
+    :param msg: error message
+    :param alternative: full text of macro as passed in MacroBase __call__
+    :param Severity: attention, caution, danger, error, hint, important, note, or tip
+    :returns: formatted text of macro, error message
     """
-    return html.div(attrib={html.class_: f"{severity} moin-nowiki"}, children=msg)
+    if severity not in "attention caution danger error hint important note tip".split():
+        severity = "error"
+        msg = (
+            _("Invalid severity, must be one of: ") + "attention, caution, danger, error, hint, important, note, or tip"
+        )
```

Comment:
does severity always come from the code or also from wiki content? if it comes from the code, guess just crashing with a ValueError would be more appropriate here (the wiki user can't fix errors in the code).

Reply:
Only from code, fixed.  Thank you.